Ecommerce_laravel_API-app/database/seeders/DatabaseSeeder.php

```diff
@@ -12,11 +12,16 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
+        $this->call([
+            ProductSeeder::class,
+        ]);
         // \App\Models\User::factory(10)->create();
```

Comment:
@parvathibhavani NEVER push commented code in GIT.. NEVER :/

Reply:
@Naveen1895 okay, here after I will not push the comment lines